src/redux/books/books.js

```diff
@@ -0,0 +1,25 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+const addBook = (book) => ({
+  type: ADD_BOOK,
+  book,
+});
+const removeBook = (book) => ({
+  type: REMOVE_BOOK,
+  book,
+});
+
```

Comment:
- [ ] **Required:** Some of your files did not have any change and are therefore not listed here which is why I am leaving this here.

![image](https://user-images.githubusercontent.com/27709832/166337982-c703f491-f921-484d-ad8f-32b1ebdd83b5.png)

Kindly fix these console errors.

- [ ] **Optional:** The add and remove functions are not functional. Books are not added to the page.

![add_remove ](https://user-images.githubusercontent.com/27709832/166338490-d8216e23-7aa1-4f45-a062-a1f1f06cd86c.gif)